DevCGlobalDirectory.md

```diff
@@ -372,6 +372,18 @@
   - Github: https://github.com/Devc-Douala
 </details>
 
+<details>
+  <summary>
+    <b>DevC Yaounde</b>
+  </summary>
+
+  - Lead(s):
+    - [Aurelien Xavier](https://github.com/xavier-pay)
+    - [Suzy Tresor Tchegnelene](#)
+  - Facebook Group: https://www.facebook.com/groups/1216588391854380/
+  - Github: #
```

Comment:
Can you remove this please if the circle doesn't have one?

Reply:
Hi, Sure! 
Will remove now because the circle doesn't have one yet, however, will update when the leads come up with one.